package-lock.json

```diff
@@ -4637,8 +4637,7 @@
             "wordwrap": {
               "version": "0.0.3",
               "resolved": "https://registry.npmjs.org/wordwrap/-/wordwrap-0.0.3.tgz",
-              "integrity": "sha1-o9XabNXAvAAI03I0u68b7WMFkQc=",
-              "dev": true
```

Comment:
Oh, wait, I think you only modified `devDependencies`. Now I really wonder what happened here.